Styles/styles.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+
+body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0px;
+}
+.background {
+    background-image: url("../Image/top-section.png");
+    height: 900px;
+    background-size: cover;
+    padding-left: 104px;
+    padding-right: 104px;
+    padding-top: 51px;
+}
+
+.trip {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.img {
+    display: flex;
+    align-items: center;
+}
+
+.img > * {
+    margin-left: 24px;
+}
+
+.link {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.a {
+    display: flex;
+}
+
+.a > * {
+    margin-right: 24px;
+    text-decoration: none;
+    color: #ffffff;
+    font-size: 24px;
+}
+
+.select {
+    border-bottom: 4px solid #F5BD41;
+    padding-bottom: 12px;
+}
+
+h1 {
+    width: 542px;
+    height: 118px;
+    font-weight: 500;
+    font-size: 50px;
+    color: #ffffff;
+    margin-top: 202px;
+}
+
+.search {
+    display: flex;
+    margin-top: 145px;
+}
+.form {
+    background-color: white;
+    display: flex;
+    border-radius: 8px;
+
+}
+.ny {
+    position: relative;
```

Comment:
It doesn't mean as there are no children of 'ny' element. Btw, what does 'ny' mean?